iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java

```diff
@@ -92,7 +92,11 @@ protected ShuntCompensatorAdder createAdder(VoltageLevel parent) {
     private static void writeModel(ShuntCompensator sc, NetworkXmlWriterContext context) throws XMLStreamException {
         if (sc.getModelType() == ShuntCompensatorModelType.LINEAR) {
             context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(context.isValid()), SHUNT_LINEAR_MODEL);
-            XmlUtil.writeDouble(B_PER_SECTION, sc.getModel(ShuntCompensatorLinearModel.class).getBPerSection(), context.getWriter());
+            double bPerSection = sc.getModel(ShuntCompensatorLinearModel.class).getBPerSection();
+            if (bPerSection == 0 && context.getVersion().compareTo(IidmXmlVersion.V_1_4) <= 0) {
+                bPerSection = Double.MIN_NORMAL;
```

Comment:
Good catch!
Can you also add a log (warning) here, explaining what is done?
This fix should also be done line 59.